packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -469,6 +469,11 @@ export class AtomicSearchInterface
         this.searchStatus.state.hasError
       );
 
+      this.host.classList.toggle(
+        'atomic-search-interface-search-executed',
+        this.searchStatus.state.firstSearchExecuted
+      );
```

Comment:
Thisone is to help displaying/hiding the recommendations when needed.

Reply:
What's the particular problem you're trying to solve with this class? Perhaps it should be something we fix within Atomic instead.

Reply:
The main issue was to show the recommendations and search results in the same modal (recs first then hide it when an initial search is executed) but not at the same time.

I was looking for the simplest way to do it without having to create an `atomic-ipx-interface` which would have been an entire copy of the search one.

After discussion with JF, he advised me to use a similar class.